epub33/overview/index.html

```diff
@@ -163,14 +210,20 @@ <h3>Reading Order</h3>
 					<h3>Navigation Document</h3>
 
 					<p>Each EPUB Publication contains a special XHTML Content Document called the <a>EPUB Navigation
-							Document</a>, which uses the [[HTML]] <code>nav</code> element to define human- and
-						machine-readable navigation information.</p>
+							Document</a>, which uses the [[HTML]] <a data-cite="html#the-nav-element"><code>nav</code></a>
+						element to define human- and machine-readable navigation information. Most Reading Systems make use of the
+						Navigation Document to present a table of content to their users.</p>
 
-					<p>The Navigation Document contains baseline accessibility and navigation support, but also features
+					<p>The Navigation Document contains baseline accessibility and navigation support, and features
 						to enhance navigation for all users. Prime among these are support for internationalization (for
-						example, as an XHTML document itself, the Navigation Document natively supports ruby
-						annotations) and support for embedded grammars (MathML and SVG can be included within navigation
-						links).</p>
+						example, as an XHTML document itself, the Navigation Document natively supports 
+						<a data-cite="html#the-ruby-element">ruby annotations</a>) and support for embedded grammars
+						(MathML and SVG can be included within navigation links).</p>
+
+					<p>Note, however, that EPUB Reading Systems are not <em>required</em> to use these advanced XHTML features, such as
```

Comment:
Minor nit, but I'd remove "however" from this sentence as the next one also starts with "However,".